factor/sms/lang/en/factor_sms.php

```diff
@@ -0,0 +1,53 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Language strings.
+ *
+ * @package     factor_sms
+ * @author      Peter Burnett <peterburnett@catalyst-au.net>
+ * @copyright   Catalyst IT
+ * @license     http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+$string['pluginname'] = 'SMS factor';
+$string['privacy:metadata'] = 'The SMS factor plugin does not store any personal data';
+$string['loginsubmit'] = 'Verify Code';
+$string['loginskip'] = "I didn't receive a code";
+$string['settings:aws:usecredchain'] = 'Use the default credential provider chain to find AWS credentials';
+$string['settings:aws:key'] = 'Key';
+$string['settings:aws:key_help'] = 'Amazon API key credential.';
+$string['settings:aws:secret'] = 'Secret';
+$string['settings:aws:secret_help'] = 'Amazon API secret credential.';
+$string['settings:aws:region'] = 'Region';
+$string['settings:aws:region_help'] = 'Amazon API gateway region.';
+$string['settings:duration'] = 'Validity duration';
+$string['settings:duration_help'] = 'The period of time that the code is valid.';
+$string['settings:countrycode'] = 'Country number code';
+$string['settings:countrycode_help'] = 'Enter the calling code for the country where user numbers reside. See this link for a list of calling codes: https://en.wikipedia.org/wiki/List_of_country_calling_codes';
```

Comment:
The calling code without the leading + as a default if users do not enter an international number with a + prefix.

See this link for a list of calling codes (+ link)